### ISSUE TITLE
A11y – Check issue with post-tabs and cross-root aria Id

### ISSUE THREAD

**Issue by imagoiq** [swisspost/design-system#2685]
Reported by cypress-axe. `post-tab-header` element has a `aria-controls` attribute with an id which references the `post-tab-panel` associated to it. However, the id is defined on an element inside the shadow-dom and is not accessible from the document root.

Some refs:

- https://nolanlawson.com/2022/11/28/shadow-dom-and-accessibility-the-trouble-with-aria/
- https://ionic.io/blog/announcing-support-for-form-associated-custom-elements-in-stencil-v4-5-0

**Comment by gfellerph** [comment]
Hoist id and aria-controls to the host element so it's part of light dom.
